pubpub/platform
# Add the context editor to forms

## Issue(s) Resolved

Closes https://github.com/pubpub/platform/pull/732
Closes https://github.com/pubpub/platform/issues/722

TODO
* [x] Validate schema from `ContextEditor` schema
* [x] Properly pass in pubs, pub types, pubId, pubTypeId, etc.
* [x] what's up with all the re-rendering? I put a workaround in the form flow, but not yet for the create/update pub flow

## High-level Explanation of PR


## Test Plan
* Add a rich text field to a pub type and to a form
* You should be able to choose the rich text component
* Fill out a form and the rich text component. Currently, only a simple paragraph will validate since I haven't imported the actual schema yet
* Save
* The editor obj should persist to the db
* Visit the newly created pub. It'll just show `[object, object]` on the pub page since that doesn't know how to render things yet, but the Update pub form should show things properly
* Creating a pub from the create pub button _should_ work too but doesn't yet

## Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/c2c7ec73-c29b-4891-99ff-f1bc1411b1aa)
![image](https://github.com/user-attachments/assets/2eb89f92-2eb7-4613-be9b-4253e6d8d5a7)
![image](https://github.com/user-attachments/assets/078f8dd4-7c27-447f-8889-012282a58d65)



## Notes
